test/stage2/arm.zig

```diff
@@ -344,4 +344,38 @@ pub fn addCases(ctx: *TestContext) !void {
             "",
         );
     }
+
+    {
+        var case = ctx.exe("integer multiplication", linux_arm);
+        // Simple while loop with assert
```

Comment:
This is not a simple while loop anymore :D

Reply:
Oops, will fix. Copy-paste errors, argh.